tests/test_parser.py

```diff
@@ -120,3 +120,13 @@ def test_annotations(self):
         assert expression.args["expressions"][3].text("this") == "c#annotation3"
         assert expression.args["expressions"][4].text("this") == "annotation4"
         assert expression.args["expressions"][5].text("this") == ""
+
+    def test_pretty_config_override(self):
+        import sqlglot
+
+        self.assertEqual(parse_one("SELECT col FROM x").sql(), "SELECT col FROM x")
+        sqlglot.pretty = True
```

Comment:
nit: you can use patch:

```python
from unittest.mock import patch

...

with patch("sqlglot.pretty", True):
    self.assertEqual(...)
```

That should prevent all subsequent tests blowing up if this fails.

Reply:
That is a great improvement. Thanks!